gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceManager.java

```diff
@@ -383,6 +386,7 @@ private void registerServicesInLauncher(){
     }
 
     this.serviceLauncher.addService(dagManager);
+    this.serviceLauncher.addService(dagProcessingEngine);
```

Comment:
always added as a svc?  as we do this (pre-)rollout and maybe even during the initial ramp, I'd expect it might be conditional based on config